default/scripting/buildings/STARLANE_BORE.focs.txt

```diff
@@ -5,27 +5,38 @@ BuildingType
     buildtime = 1 + Target.System.NumStarlanes
     location = Planet
     effectsgroups = [
+        [[SPECIES_LIKES_OR_DISLIKES_BUILDING_STABILITY_EFFECTS]]
+
+        EffectsGroup
+            scope = Object id = Source.SystemID
+            activation = Not [[ BORE_POSSIBLE ]]
```

Comment:
There is still a minor issue here: If multiple bores trigger in the same turn, all report success, even if some of them could not create a new lane.

Reply:
probably less of an issue than what it's replacing...